src/config/updateExtremes.js

```diff
@@ -62,9 +64,36 @@ const updateExtremesYAxis = (axis, contract, ticks) => {
     }
 };
 
-export default (chart, ticks, contract) => {
+const updateExtremes = (chart, ticks, contract) => {
     if (!contract) return;
 
-    updateExtremesXAxis(chart.xAxis[0], contract);
-    updateExtremesYAxis(chart.yAxis[0], contract, ticks);
+    const xAxis = chart.xAxis[0];
+    const xMin = xAxis.min;
+    const xMax = xAxis.max;
+
+    const zoomedTicks = ticks
+        .filter(t => (t.epoch * 1000) > xMin && (t.epoch * 1000) < xMax);
+
+    let ticksMin = 0;
```

Comment:
We certainly do want to control it.
When you show a statement and open contract.